x: 14.27
y: 31.94
th: 0
cov_x: 0.0058
cov_y: 0.00087
cov_th: 0.0015
